.github/workflows/auto-measure.yml

```diff
@@ -0,0 +1,31 @@
+name: Create Release and Publish Files
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  publish:
+    runs-on: ubuntu-20.04
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - name: Make snp.sh executable and run
+      run: |
+        sudo chmod +x snp.sh
+        sudo ./snp.sh build-guest --non-upm
+
+    - name: Make cal-measurement.sh executable and run
+      run: |
+        sudo chmod +x cal-measurement.sh
+        ./cal-measurement.sh
+
+    - name: Upload files to a GitHub release
+      uses: svenstaro/upload-release-action@2.7.0
+      with:
+        repo_token: ${{ secrets.GITHUB_TOKEN }}
+        file: ./output.txt
+        asset_name: output.txt
+        tag: ${{ github.ref }}
```

Comment:
Have you test this script?

Reply:
I have not. I am now testing [here](https://github.com/jingyao-zhang/snp-auto-measurement/actions). 
It may take a while. The potential problem would be `You are running out of disk space`. 
Let's see!

Reply:
It seems github action server cannot run the script since it is necessary to launch a guest VM and then replace the guest kernel by the newly built guest kernel. 
However, the github action server does not support KVM. 
I am searching for alternative methods.

Reply:
@Xynnn007 The github action works fine now with the new commits in this PR. 
There is an extra effort on the repo owner side to enable successful github action. The repo's `Settings -> Actions -> General -> Workflow permissions -> Read and write permissions` is required to be **enabled** to let the workflow write the generated measurement into the repo release.